Makefile

```diff
@@ -14,7 +14,7 @@ stop_s3:
 	docker stop `docker ps -aqf "name=voogle-s3"`; docker rm voogle-s3
 
 start_redis:
-	docker run --name redis-test-instance -p 6379:6379 -d redis
+	docker run -d --name redis-cluster -e ALLOW_EMPTY_PASSWORD=yes -e REDIS_NODES=3 -p 6379:6379 bitnami/redis-cluster:6.2
 
 stop_redis:
-	docker stop redis-test-instance; docker rm redis-test-instance
+	docker stop redis-cluster; docker rm redis-cluster
```

Comment:
Instead of using those docker command, maybe we should only use the docker-compose and add the command in the Makefile